src/refg.cpp

```diff
@@ -68,9 +69,52 @@ refg_t refg_registry_t::as_refg(std::string_view config) const
 	return ref;
 }
 
+std::string_view refg_registry_t::_try_refg_as_sv_from_file(refg_t ref) const
```

Comment:
would be good to have a test

Reply:
I'm not sure how to implement a test w/o first loading the relevant Genome (thus making the test kinda meaningless).

Best I can come up with is to add a small binary file (e.g gtrack) to the repo, but I'd rather not.